File: ProjectEuler/thousand=0/hundred=00/ten=000/problem_0005.py
# https://projecteuler.net/problem=5
def solution(lower, upper):
    value = upper

    while True:
        for divisor in range(lower, upper + 1):
            if(value % divisor != 0):
                break;
        else:
            return value

        value += upper

print(solution(1,20))
# 232792560

File: ProjectEuler/clojure/thousand=0/hundred=00/ten=000/problem_0005.clj
(ns project-euler.p.0005)

(defn evenly-divisible
  ([lower upper] (evenly-divisible lower upper upper))
  ([lower upper value]
   (if (every?
         #(= 0 (rem value %)) 
         (range lower (inc upper)))
     value
     (recur lower upper (+ value upper)))))
  
(evenly-divisible 1 20)
